lib/elf_py.c

```diff
@@ -1319,6 +1319,8 @@ static PyObject *elffile_load(PyTypeObject *type, PyObject *args,
 	if (w->elf)
 		elf_end(w->elf);
 	free(w->filename);
+	if (fd)
```

Comment:
the file descriptor is closed soon after opening and will never be open down here, so this change should be removed; however, at line 1143:
```
		if (fd > 0)
			close(fd);
```
technically should be
```
		if (fd >= 0)
			close(fd);
```
because 0 is a valid file descriptor.